sdk/react/constants.js

```diff
@@ -129,6 +129,8 @@ const VIEW_ACCESSIBILITY_NAMES = {
   CC_CELL: 'Subtitle cell. Tap twice to choose this subtitles',
   PROGRESS_BAR: 'Progress bar. Use two fingers to adjust the progress value',
   VOLUME_BAR: 'Volume bar. Use two fingers to adjust the volume value',
+  FORWARD_BUTTON: 'Forward button. Press twice to seek forward on ',
```

Comment:
Small change: take out the "on" and the extra spaces at the end. For both of these descriptions.

Reply:
@pilievOoyala This space is required to make a composite string. In accessibility it will be something like: `Forward button. Press twice to seek forward on 10 seconds`

Reply:
@IvanSakharovskii, we could add the space like this: `'Press twice to seek forward on' + '  ' + x + 'seconds'`. 

I assume that right now it is `'Press twice to seek forward on ' + x + 'seconds'`.

Understand why you want the space, not sure which is the right approach, but yea would suggest to keep the Constants without spaces.

At least for now remove the "on" :)

Reply:
@rcvrgs Ok, done :)